src/custom/pages/Claim/CanUserClaimMessage.tsx

```diff
@@ -26,7 +26,10 @@ export default function CanUserClaimMessage({ hasClaims, isAirdropOnly }: ClaimI
           <Trans>
             Thank you for being a supporter of CowSwap and the CoW protocol. As an important member of the CowSwap
             Community you may claim vCOW to be used for voting and governance. You can claim your tokens until{' '}
-            <i>{airdropDeadline && new Date(airdropDeadline).toLocaleString()}</i>
+            <i>
+              {airdropDeadline && new Date(airdropDeadline).toLocaleString()} (
+              {Intl.DateTimeFormat().resolvedOptions().timeZone})
```

Comment:
Should we move this to a function in `utils/time.ts` that would do the whole formatting? 
`formatDateWithTimezone(date):string`?

Reply:
Yes probably cleaner as we could use it in the claim table as well!

Reply:
https://github.com/gnosis/cowswap/pull/2239